BUILD

```diff
@@ -1,10 +1,20 @@
-#Simple Bazel build file that will be replaced with the actual fuzz target rule functionality.
-load("@rules_cc//cc:defs.bzl", "cc_binary")
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
 
-cc_binary(
-    name = "hello_world",
-    srcs = ["hello_world.cc"],
-)
+# Simple Bazel build file that will be replaced with the actual fuzz target rule functionality.
 
 cc_test(
```

Comment:
I still think this is a candidate for removal, as I'm not sure it serves any specific purpose now that we have fuzz targets checked in as examples.

Reply:
I removed this dummy_test.cc before, but the CI requires at least one test exists. For the fuzz targets in examples, they either return error or get timeout. I am wondering if you have any way to suppress the error?

Reply:
> but the CI requires at least one test exists.

I believe it was just the `bazel test` command that was requiring at least one test to exist - but we can remove that command and only keep the one that runs the fuzz target builds. (In general, I believe we should adjust our CI scripting to fit our repository structure, not the other way around.)